src/GraphQL.Harness/GraphQLMiddleware.cs

```diff
@@ -63,7 +63,6 @@ private async Task ExecuteAsync(HttpContext context, ISchema schema)
                 options.Inputs = request.Variables.ToInputs();
                 options.UserContext = _settings.BuildUserContext?.Invoke(context);
                 options.EnableMetrics = _settings.EnableMetrics;
-                options.ExposeExceptions = _settings.ExposeExceptions;
```

Comment:
The breaking change is, mainly, to remove this line of code and instead register an `IErrorParser` within the DI framework